src/config/Config.ts

```diff
@@ -167,7 +170,7 @@ const DEFAULT_PROPERTIES: Properties = {
     'hazelcast.invalidation.reconciliation.interval.seconds': 60,
     'hazelcast.invalidation.max.tolerated.miss.count': 10,
     'hazelcast.invalidation.min.reconciliation.interval.seconds': 30,
-    'hazelcast.logging.level': 'INFO',
+    'hazelcast.logging.level': LogLevel.INFO,
```

Comment:
I think you should revert this. The default value should be `INFO`, not `2`

Reply:
no this is correct. The default properties should have the same types with config builder's return type so an integer(or enum). if logging level property is not given and its default value is used, logging level is directly passed to the default logger which expects an integer.  

otherwise these fail:
```js
it('default logging in case of empty property', async function () {
        client = await Client.newHazelcastClient({
            clusterName: cluster.id
        });
        sinon.assert.called(console.log);
    });
```
```js
it('default logging in case of empty property', async function () {
        client = await Client.newHazelcastClient({
            clusterName: cluster.id
        });
        sinon.assert.called(console.log);
    });
```

Reply:
Hmm, the default value is `2` (`LogLevel.INFO`), but if you pass something like 
```
newHazelcastClient({properties: {"hazelcast.logging.level": LogLevel.INFO}}) 
```
(or whatever the correct thing is :smile: ), it will fail, right?

Reply:
DEFAULT_PROPERTIES is internal isn't it? the user knows that default value is 'INFO', not LogLevel.INFO. we document it like this. Using enums is harder for users, so we decided to go with strings in the past.

Reply:
OK, I am not really enjoying this but I can live with that